### ISSUE TITLE
Tag cloud layout creates word cloud visualization

### ISSUE THREAD

**Issue by jaguillette** [anvc/scalar#190]
When selecting a "Tag cloud" layout option for a page, the visualization created at the top of the page is a word cloud, although I'm not certain where the word cloud data is drawn from. The interaction and tooltip is the same as a visualization widget word cloud, and no longer contains links to tags. This was reported by a faculty member, and I don't yet know when the last time that the visualization worked as expected was, but I'll update this issue when I find out.

**Comment by jaguillette** [comment]
It seems that the "tag cloud" layout was working in December, so it's likely a change since then.

**Comment by eloyer** [comment]
Can confirm, we're looking into a remedy.

**Comment by jaguillette** [comment]
That's correct! We're due for an update from the main branch anyways, so we'll be updating this sprint. Thank you!
